src/ResourceManager/Network/Commands.Network/help/Get-AzureRmNetworkWatcherConnectionMonitor.md

```diff
@@ -0,0 +1,230 @@
+---
```

Comment:
@irrogozh it looks like the [module page](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/help/AzureRM.Network.md) was not updated with the new cmdlets. Would you mind adding them to this file so users could easily find them?

Reply:
Done